addons/l10n_ro/account_tax_template.xml

```diff
@@ -1,9 +1,494 @@
 <?xml version="1.0" encoding="utf-8"?>
 <openerp>
 <data noupdate="0">
+    
+    <record id="tvati" model="account.tax.template">
```

Comment:
Is it an update of the [original tvati tax](https://github.com/odoo/odoo/blob/dbb0b42695b987894a73d0b78a8c9de43fefac28/addons/l10n_ro/account_tax_template.xml#L184) or a new one? Be careful that people updating their odoo will still get the old values for the fields you have not specified like `ref_tax_code_id` or `account_collected_id`